src/calendar-list/item.tsx

```diff
@@ -101,6 +101,7 @@ class CalendarListItem extends Component<CalendarListItemProps, CalendarListItem
       headerStyle,
       onPressArrowLeft,
       onPressArrowRight,
+      //@ts-ignore
```

Comment:
Consider using `// @ts-expect-error` if this will be fixed eventually.
Maybe once we'll move other files this will resolve itself and it will notify us about it

Reply:
Done